contribute/CONTRIBUTE-CODE.rst

```diff
@@ -0,0 +1,934 @@
+.. Licensed to the Apache Software Foundation (ASF) under one
+    or more contributor license agreements.  See the NOTICE file
+    distributed with this work for additional information
+    regarding copyright ownership.  The ASF licenses this file
+    to you under the Apache License, Version 2.0 (the
+    "License"); you may not use this file except in compliance
+    with the License.  You may obtain a copy of the License at
+
+ ..   http://www.apache.org/licenses/LICENSE-2.0
+
+ .. Unless required by applicable law or agreed to in writing,
+    software distributed under the License is distributed on an
+    "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+    KIND, either express or implied.  See the License for the
+    specific language governing permissions and limitations
+    under the License.
+
+.. contents:: :local:
+
+Welcome
+=============
+
+Welcome to Apache Airflow! Contributions from the community are the heartbeat of the project. Every little bit helps,
+and credit will always be given.
+
+The guide below contains instructions on how to contribute to core Apache Airflow code. If you are a new contributor,
+follow `Contributors Quick Start <https://github.com/apache/airflow/blob/master/contribute/CONTRIBUTE_QUICK_START.rst>`__ first for easy, step-by-step guidelines on how to set up your development
+environment and make your first contribution.
+
+If you're interested in contributing to other parts of the project, refer to
+`Contribute to Airflow Documentation <https://github.com/apache/airflow/blob/master/contribute/CONTRIBUTE-DOCS.rst>`__ or `Contribute to Airflow Providers <https://airflow.apache.org/docs/apache-airflow-providers/index.html#creating-your-own-providers>`__.
+
+Where to Contribute
+--------------------
+To get started, decide where or what you want to contribute. For ideas, you can browse:
+
```

Comment:
- `Good First Issues <https://github.com/apache/airflow/issues?q=is%3Aopen+is%3Aissue+label%3A%22good%20first%20issue%22>`__

Reply:
Good call here too - added